tests/lean/interactive/run.lean

```diff
@@ -107,6 +107,9 @@ partial def main (args : List String) : IO Unit := do
           for diag in diags do
             IO.eprintln (toJson diag.param)
           requestNo := requestNo + 1
+        | "collectInitDiagnostics" =>
+          for diag in initDiags do
+            IO.eprintln (toJson diag.param)
```

Comment:
Is this okay? It looks like we were discarding the initial diagnostics (that is, we were expecting some change to the file before collecting diagnostics), so I added this command so we can print them out anyways if we want to. But perhaps the interactive tests is not the best place to put this workspace symbol search test?

Reply:
> But perhaps the interactive tests is not the best place to put this workspace symbol search test?

Exactly, you can put it in `tests/lean` with the other output tests